src/CMakeModules/Bootstrap_Linux.cmake

```diff
@@ -59,38 +58,19 @@ FIND_PACKAGE (X11 REQUIRED)
 
 if (ENABLE_HTML)
     if (USE_SYSTEM_CEF)
-        set(CEF_LIB_PATH "/usr/lib/casparcg-cef-117")
```

Comment:
with this being `find_package(CEF 117 REQUIRED)`, what would happen if I had `libcef117` and `libcef138` installed on my machine? would it be able to find the correct one?